src/common/harvest/harvest.js

```diff
@@ -160,6 +160,9 @@ export class Harvest extends SharedContext {
       }, eventListenerOpts(false))
     }
 
+    const runtime = getRuntime(this.sharedContext.agentIdentifier)
+    runtime.harvestCount++
```

Comment:
I was going to ask about making this a class private since everything has to come through this module but this makes sense because we dont want this module having to worry about the url/data it is transmitting.